backend/typescript/graphql/index.ts

```diff
@@ -67,7 +71,7 @@ const graphQLMiddlewares = {
     simpleEntities: authorizedByAllRoles(),
     userById: authorizedByAdmin(),
     userByEmail: authorizedByAdmin(),
-    users: authorizedByAdmin(),
+    // users: authorizedByAdmin(),
```

Comment:
I'm not sure if this was for testing purposes, but I think we should have this uncommented at least in the main dev branch